docs/intro/intro_quadratic.rst

```diff
@@ -15,31 +15,63 @@ When using such samplers to handle problems with constraints, you typically
 formulate the constraints as penalties: see
 :std:doc:`sysdocs_gettingstarted:doc_getting_started`.
 (:ref:`Constrained models <intro_cqm>`, such as the
-:class:`ConstrainedQuadraticModel`, can support constraints natively.)     
+:class:`dimod.ConstrainedQuadraticModel`, can support constraints natively.)
 
-Quadratic Models
+Supported Models
 ================
 
-dimod provides a :term:`quadratic model` (QM) class
+* **Quadratic Models**
 
-Binary Quadratic Models
-=======================
+  The :term:`quadratic model` (QM) class, :class:`dimod.QuadraticModel`, encodes
+  polynomials of binary, integer, and discrete variables, with all terms of degree
+  two or less.
 
-dimod provides a :term:`binary quadratic model` (BQM) class that encodes
-:term:`Ising` and quadratic unconstrained binary optimization (\ :term:`QUBO`\ )
-models used by samplers such as the D-Wave system.
+  For the QM class, its attributes and methods, see :ref:`qm`.
 
-For an introduction to BQMs, see :std:doc:`Binary Quadratic Models <oceandocs:concepts/bqm>`.
+* **Binary Quadratic Models**
 
-dimod can represent BQMs with classes suited to different needs for
-performance and ease of use. Beginners might start with the simple Python
-:ref:`AdjDictBQM <adjdictbqm_dimod>` class (:class:`dimod.BinaryQuadraticModel`),
-and later switch to a higher-performing class with a C++ implementation.
+  The :term:`binary quadratic model` (BQM) class, :class:`dimod.BinaryQuadraticModel`,
```

Comment:
why not make the term a link to the glossary? Or does that happen automatically

Reply:
The `:term:` creates a link to the glossary. You can see the mutual links in the glossary terms themselves here: https://docs.ocean.dwavesys.com/en/stable/concepts/index.html
An `binary quadratic model` is the first glossary entry